examples/distributed-training/trl/README.md

```diff
@@ -0,0 +1,184 @@
+# TRL
+
+This example walks you through how to run distributed fine-tune using [TRL](https://github.com/huggingface/trl), [Accelerate](https://github.com/huggingface/accelerate) and [Deepspeed](https://github.com/deepspeedai/DeepSpeed) with `dstack`.
+
+??? info "Prerequisites"
+    Once `dstack` is [installed](https://dstack.ai/docs/installation), go ahead clone the repo, and run `dstack init`.
+
+    <div class="termy">
+ 
+    ```shell
+    $ git clone https://github.com/dstackai/dstack
+    $ cd dstack
+    $ dstack init
+    ```
+    </div>
+
+## Create fleet
+
+Before submitted disributed training runs, make sure to create a fleet with a `placement` set to `cluster`.
+
+> For more detials on how to use clusters with `dstack`, check the [Clusters](https://dstack.ai/docs/guides/clusters) guide.
+
+## Run Distributed Training
+Once the fleet is created, define a distributed task configuration. Here's an example of distributed Supervised Fine-Tuning (SFT) task using `FSDP` and `Deepseed ZeRO-3`.
+
+
+=== "FSDP"
+
+    <div editor-title="examples/distributed-training/trl/fsdp.dstack.yml">
+    ```yaml
+    type: task
+    # The name is optional, if not specified, generated randomly
+    name: trl-train-fsdp-distrib
+
+    # Size of the cluster
+    nodes: 2
+
+    image: nvcr.io/nvidia/pytorch:25.01-py3
+
+    # Required environment variables
+    env:
+      - HF_TOKEN
+      - ACCELERATE_LOG_LEVEL=info
+      - WANDB_API_KEY
+      - MODEL_ID=meta-llama/Llama-3.1-8B
+      - HUB_MODEL_ID
```

Comment:
What is `HUB_MODEL_ID` environment variable?  How is it diffrenent in this context from `MODEL_ID`. Why does the Axolotl examample, use `HUB_MODEL_ID=meta-llama/Meta-Llama-3-70B`?